### PR TITLE
Vinayapriya C C

### PR DESCRIPTION
**Pull request by Vinayapriya**

### CODE REVIEW

File: participants/template.md
Comment (rajatsandeepsen) [inline participants/template.md:1]: Dont delete this file

**Review by rajatsandeepsen — CHANGES_REQUESTED** [review]
Cool ✨

File: participants/template.md
Comment (Vinayapriya) [reply participants/template.md:1]: updated

**Comment by alvin-dennis** [comment]
@Milansuman u can merge this after ur final recheck.

**Pull request merged**